chapters/standard_subtypes/cameras.txt

```diff
@@ -30,12 +28,23 @@ All cameras sources accept the following parameters:
 |===================================================================================================
 
 The camera is placed and oriented in the world with `position`,
-`direction` and `up`. Additionally, an extra transformation `transform`
-can be specified, which will only be applied to 3D vectors (i.e.
-`position`, `direction` and `up`), but does _not_ affect any sizes
+`direction` and `up`. Additionally, an extra affine transformation
+`transform` can be specified, which applied to homogeneous coordinates.
```

Comment:
Nit: "...which applied to homogeneous coordinates" should be "...which applies to homogeneous coordinates"

Reply:
thanks, fixed